package.json

```diff
@@ -6,11 +6,14 @@
   "dependencies": {
     "@ctrl/tinycolor": "3.5.0",
     "@tsconfig/svelte": "3.0.0",
+    "@types/dompurify": "^2.4.0",
+    "dompurify": "^2.4.1",
+    "jsdom": "^20.0.3",
```

Comment:
Everything else in dependencies has the version pinned, so seems like we should continue to do that for these new ones